docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -430,7 +430,7 @@ In the table above, whenever `NotificationResponseReceivedListener` is triggered
 
 > **warning** Background event listeners are not supported in Expo Go.
 
-To handle notifications while the app is backgrounded on iOS, you _must_ add `remote-notification` to the `ios.infoPlist.UIBackgroundModes` key in your **app.json**,
+To handle notifications while the app is backgrounded on iOS, you _must_ add `remote-notification` as an element in an array within the `ios.infoPlist.UIBackgroundModes` key in your **app.json**,
```

Comment:
```suggestion
To handle notifications while the app is in the background on iOS, add `remote-notification` as a value to the array under `ios.infoPlist.UIBackgroundModes` key in your [app config](/versions/latest/config/app/),
```

- Clearly say that `remote-notification` is value for the array under key
- Instead of saying app.json, we can point towards app config and provide a link (in some cases the file can be app.config.js)
- Remove passive voice usage (unrelated to this PR)